packages/dev/core/src/Culling/ray.ts

```diff
@@ -798,7 +801,8 @@ Scene.prototype._internalPick = function (
             continue;
         }
 
-        const world = mesh.getWorldMatrix();
+        const forceCompute = multiCamera && mesh.isWorldMatrixCameraDependent();

```

Comment:
We could also add `&& this.cameraToUseForPointers !== this.activeCamera` to avoid recomputing the matrix if it's already ok (maybe add this code in `multiCamera` instead, to avoid doing it in the loop).